### ISSUE TITLE
Identify why some Bridgy posts seems to get duplicated

### ISSUE THREAD

**Issue by voxpelli** [voxpelli/webpage-webmentions#35]
First [pointed out](https://chat.indieweb.org/2016-06-24/1466806062067000) by @kevinmarks on: http://www.kevinmarks.com/decentralweb2016-06-08.html

Then also discovered on @miklb's blog: https://miklb.com/jekyll-and-microblogging

This happened before the deploy of `0.11.x` so it means that the cause has to be outside of this service, but it could very well be due to me interpreting something wrong. Will have to dig into it.

**Comment by voxpelli** [comment]
Miklb's blog doesn't use my endpoint which point further towards it being eg. Bridgy that causes this. When debugging I'm seeing some weird behaviors though where different data is returned between refreshes, so will continue investigation.

**Comment by kevinmarks** [comment]
I do sometimes see what would be facepiles show up as individual posts
instead.

**Comment by voxpelli** [comment]
@kevinmarks Yes, I just identified a caching issue where a cache which was supposed to prevent outages if the database were to be gone for a few seconds managed to mix up entry responses with each other. Deploying a fix for that right now.

**Comment by voxpelli** [comment]
In regards to the duplicates – Bridgy has sent these:

* https://brid-gy.appspot.com/post/twitter/kevinmarks/741827388313456640
* https://brid-gy.appspot.com/post/twitter/kevinmarks/741826520285470720
* https://brid-gy.appspot.com/post/twitter/kevinmarks/741826676758216705

+ a couple of more

They were all sent a minute apart from each other and the mf2 data within them are exactly the same:

* http://pin13.net/mf2/?url=https://brid-gy.appspot.com/post/twitter/kevinmarks/741827388313456640
* http://pin13.net/mf2/?url=https://brid-gy.appspot.com/post/twitter/kevinmarks/741826520285470720

The way to get around these duplicates would be to dedupe based on the `u-url` of the `h-entry`, but that would in a way be a Brid.gy specific fix as such a dedupe would enable a third party to block a true Webmention by being the first to ping with a specific `u-url`, no matter of it owns that url or not.

I'm opening an issue in Bridgy about this as it should be solved there as well anyhow.

**Comment by voxpelli** [comment]
Related wiki brainstorming around de-duplication: https://indieweb.org/Webmention-brainstorming#de-duplication
